modules/ROOT/examples/live-demos/advtemplate-mce-clipboard/index.html

```diff
@@ -0,0 +1,29 @@
+<textarea id="advanced-template-mce-clipboard">
+  <h3>Using the mce-clipboard Marker with Advanced Templates<h3>
```

Comment:
```suggestion
  <h3>Using the {{mce-clipboard}} Marker with Advanced Templates<h3>
```

Reply:
I'll retest this change suggestion locally, but i believe that using `{{mce-clipboard}}` was not rendering in the demo, so i removed the `{{}}`